.github/workflows/ci-ipfs.yml

```diff
@@ -0,0 +1,83 @@
+name: Deploy IPFS Base
+
+on:
+  workflow_call:
+    inputs:
+      current_branch:
+        description: "Current branch"
+        required: true
+        type: string
+      ipfs_folder:
+        description: "IPFS build folder"
+        required: true
+        type: string
+      W3S_TOKEN:
+        description: "web3storage token"
+        required: true
+        type: string
+      GW3_TOKEN:
+        description: "gateway3 token"
+        type: string
```

Comment:
Shouldn't it be a secrets?

Reply:
@choooze 
it should be in repository secrets. But in this workflow these are just input parameters. How secrets will be managed depends on the team that will use this workflow.
I will write a guide on how to use this workflow